dependencyManagement/build.gradle.kts

```diff
@@ -12,7 +12,7 @@ val dependencyVersions = hashMapOf<String, String>()
 rootProject.extra["versions"] = dependencyVersions
 
 val DEPENDENCY_BOMS = listOf(
-  "com.fasterxml.jackson:jackson-bom:2.14.0",
+  "com.fasterxml.jackson:jackson-bom:2.14.1",
```

Comment:
Huh
https://github.com/FasterXML/jackson-bom/blob/98a2f3d479ed786afc7eefc963b01eb93a83a95a/release-notes/VERSION-2.x#L16-L18

Reply:
Looks like the changelog only details bom changes, which would still mean that artifacts managed by the bom could have changes, such as [jackson-databind](https://github.com/FasterXML/jackson-databind/blob/2.15/release-notes/VERSION-2.x#L12-L20).